README.md

```diff
@@ -50,6 +50,7 @@ English | [한국어](./README-KR.md)
 - [avante.nvim](https://github.com/yetone/avante.nvim) - Neovim plugin designed to emulate the behavior of the Cursor AI IDE. It provides AI-driven code suggestions and allows you to apply recommendations directly to your source files with minimal effort.
 - [backnotprop/prompt-tower](https://github.com/backnotprop/prompt-tower) - A tool that helps you build prompts with many code blocks.
 - [Augment Code](https://www.augmentcode.com/) - A AI coding assistant built for professional software engineers and large codebases.
+- [Ridvay Code](https://marketplace.visualstudio.com/items?itemName=Ridvay.ridvay-code) - AI-powered coding agent for VS Code, manipulates files, runs commands, automates browsers, and adapts through Custom Modes to be your flexible development partner for more efficient software building.
```

Comment:
Nice. If I were you, I would remove  "AI-powered" as all tools here are.

This is how I will pitch it or how I will buy in where I to read:

> The most efficient development coding partner capable of manipulating files, execute commands, browse the net right inside VS Code.

Or something in those lines.